app/services/proofing/aamva/applicant.rb

```diff
@@ -11,6 +11,10 @@ def self.from_proofer_applicant(applicant)
           last_name: applicant[:last_name],
           dob: format_dob(applicant[:dob]),
           state_id_data: format_state_id_data(applicant),
+          address1: applicant[:address1],
+          city: applicant[:city],
+          state: applicant[:state],
+          zipcode: applicant.fetch(:zipcode, '')[0..4],
```

Comment:
the `.fetch()` means we'll have an empty string in the case when there's `nil` for city, state, etc, is that what we want? What if we used our friend `.&`?:

```suggestion
          zipcode: applicant[:zipcode]&.slice(0..4)
```

Reply:
We can use one or the other. I'm not sure there is a meaningful difference between empty string and nil here.